docs/tasks.md

```diff
@@ -137,7 +137,7 @@ Below is an example of a Pipeline declaration that uses a `ClusterTask`:
 - The cluster resolver syntax below can be used to reference any task, not just a clustertask.
 
 {{< tabs >}}
-{{< tab header="v1 & v1beta1" code=true lang="yaml" >}}
+{{% tab header="v1 & v1beta1" code=true lang="yaml" %}}
```

Comment:
With this change are the additional attributes from https://github.com/tektoncd/pipeline/pull/6676 still required?

Or could we clean this up too?

```diff
-{{% tab header="v1 & v1beta1" code=true lang="yaml" %}}
+{{% tab "v1 & v1beta1" %}}
```

Reply:
I think it needs a cleanup. Perks of search and replace.  I will fix it ASAP.